src/pages/Home.tsx

```diff
@@ -1,9 +1,16 @@
-import { useState } from 'react';
+import { useState,useEffect } from 'react';
 import StockViewer from '../components/StockViewer';
 import Listing from '../pages/StockListing/StockListing';
 const Home = (): JSX.Element => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
-
+  
+  useEffect(() => {
```

Comment:
Storing the logged in state like this in `localStorage` works, but it might be possible for the user to somehow manipulate this piece of state, and I'm not sure if storing and getting the state is somehow creating another bug (see comments below) where we keep making requests to `/api/v1/stocks` in an infinite loop. It might be better to determine the `loggedIn` state by making a GET request to the backend at `/api/v1/authenticated`, and passing that down as state or context. However, if we did that, I think we should do that part after we also merge Regina's PR on routing.

Reply:
Isn't the backend sending a JWT?

Why don't check if the user is login or not with it?